docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1568 @@
+{
```

Comment:
<p>Replace "1st" with "first."</p><p> </p><p>Also I think it's worth writing "user-defined function (UDF)" here. That abbreviation is not very common, in my opinion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>